help/C/Help_ch_Transactions.xml

```diff
@@ -899,180 +905,543 @@ Author:
+          </listitem>
+          <listitem>
+            <emphasis role="bold"><guilabel>Tradable commodities</guilabel></emphasis>
+            <para>Not yet implemented</para>
+          </listitem>
+          <listitem>
+            <emphasis role="bold"><guilabel>QIF Import</guilabel></emphasis>
+            <para>Click on <guibutton><emphasis role="bold">Start</emphasis></guibutton> to begin the import. &app; 
+              currently proceeds to the Update your &app; accounts tab if the import is successful.
+            </para>
+          </listitem>
+          <listitem>
+            <emphasis role="bold"><guilabel>Match existing transactions</guilabel></emphasis>
+            <para>Not yet implemented.</para><!--FIXME when implemented.-->
+          </listitem>
+          <listitem>
+            <emphasis role="bold"><guilabel>Select possible duplicates</guilabel></emphasis>
+            <para>Not yet implemented</para><!--FIXME when implemented.-->
+          </listitem>
+          <listitem>
+            <emphasis role="bold"><guilabel>Update yourGnuCash accounts</guilabel></emphasis>
+            <para>Click <guibutton><emphasis role="bold">Apply</emphasis></guibutton> 
+              to complete the import. Depending upon the size of your file, the import might take a few minutes to complete,
+              so a progress bar displays the percentage finished. When the import process is complete, &app; will return you 
               to the main window, which should now display the names of the accounts you imported.
+            </para>
+            <para>Other features are not currently implemented.<!--FIXME when implemented.--></para>
+          </listitem>
+          <listitem>
+            <emphasis role="bold"><guilabel>QIF Import Summary</guilabel></emphasis>
+            <para>Click on <guibutton><emphasis role="bold">Close</emphasis></guibutton> to exit the wizard.</para>
+          </listitem>
+          </itemizedlist>
         </para>
       </sect3>
     </sect2>
+    
     <sect2 id="trans-import-ofx">
       <title>Import OFX/QFX</title>
-      <para>This opens a file selection dialog. Navigate to the file you wish to import, select a file with 
-        the appropriate extension (.ofx  or .qfx), then press the <guibutton>Import</guibutton> button button.</para>
-      <para>Gnucash opens an Account Selection dialog to select an account in your CoA corresponding to data source. Select
-        the appropriate account from the account tree and press the <guibutton>OK</guibutton> button. On subsequent import of files from the 
-        same source (identified by tags in the file), the source is remembered and the account selection dialog is not
-        displayed.</para>
+      <para><guimenu>File</guimenu>-><guisubmenu>Import</guisubmenu>-><guimenuitem>Import OFX/QFX transactions</guimenuitem>
```

Comment:
Should we explain OFX and QFX? Again, here or in the glossary.

Reply:
It would be nice to link to external definitions but the volatility of that is not controllable. My personal preference is for explaining formats where they are used unless we can link directly into the glossary. What would be nice  would be to have sections from a glossary appear as popups when you hover on them, at least in the html if not the other formats.  I was not familiar with glossries in DocBook but it appears you can have them linked from the text.  I will have to experiment. Is there currently a glossary implemented in the documentation? Would you envisage the glossary to be attached to a chapter or to be documentation wide? I gather both are possible http://www.sagehill.net/docbookxsl/GlossDatabase.html. AFAI can see at the moment we don't have a general glossary but it would obviously be useful when it comes to terms like "split" which have a GnuCash specific meaning as well as the many acronyms which come up which could all be defined in a document.  It would appear that you have to setup an XML  catalog to do the most general approach for a multidocument gloassary..

Reply:
I would consider wikipedia as trustworthy and stable. I defined several base URLs in docbook/gnc-docbookx.dtd.
To link https://en.wikipedia.org/wiki/QFX_(file_format), you can use
`<ulink url="&url-wp-en;QFX_(file_format)" />`

@sunfish62 created a glossary in guide. IMHO as a referential element it would belong into help, but until we fix linking between the docs you could use 
`<ulink url="&url-docs-build-C;gnucash-guide/gnc-gloss.html" />`

Reply:
Perhaps setting up a gnucash XML catalog file in the docbook directory of the sources is the way to make the @sunfish62 glossary available to all documents which would be good. Then if entries have to be edited it only has to happen in one place. Once it exists it would require creation of an environment variable XML_CATALOG_FILES, if it doesn't already exist and appending the URL to it if it does, so xsltproc can locate it during processing. I assuming in ignorance that  we are using xsltproc and not one of the alternatives. That way the glossary is independent of both the guide and help and available in both.  I will put all the other corrections in first  then come back and experiment with making the glossary more general. @sunfish62 may have some suggestions re this as well as he more than likely had this in mind when creating the glossary. The glossary already  has entries for the various import export formats already except for DTAUS and MT942. I noticed David T used link ids so doing a global replace  of <text>with <glossterm OFX linkend="gnc_gl_ofx"><text></glossterm>  will fix allow using the glossary in the help file.